src/Authoring/WinRT.SourceGenerator/WinRTRules.cs

```diff
@@ -18,124 +18,125 @@ private static DiagnosticDescriptor MakeRule(string id, string title, string mes
-            "WME",
+            "CsWinRT1005",
             "Class is unsealed",
-            "Exporting unsealed types is not supported. Please mark type {0} as sealed.");
+            "Exporting unsealed types is not supported in CsWinRT. Please mark type {0} as sealed.");
 
         public static DiagnosticDescriptor UnsupportedTypeRule = MakeRule(
-            "WME",
+            "CsWinRT1006",
             "Exposing unsupported type",
-            "The member '{0}' has the type '{1}' in its signature. The type '{1}' is not a valid Windows Runtime type\n"  
-            + "Yet, the type (or its generic parameters) implement interfaces that are valid Windows Runtime types\n" 
-            + "Consider changing the type '{1} in the member signature to one of the following types from System.Collections.Generic:\n{2}");
+            "The member '{0}' has the type '{1}' in its signature. The type '{1}' is not a valid Windows Runtime type. "  
+            + "Yet, the type (or its generic parameters) implement interfaces that are valid Windows Runtime types. " 
+            + "Consider changing the type '{1} in the member signature to one of the following types from System.Collections.Generic: {2}.");
         
         public static DiagnosticDescriptor StructWithNoFieldsRule = MakeRule(
-            "WME1060",
+            "CsWinRT1007",
             "Empty struct rule",
             "Structure {0} contains no public fields. Windows Runtime structures must contain at least one public field.");
  
         public static DiagnosticDescriptor NonWinRTInterface = MakeRule(
-            "WME1084",
+            "CsWinRT1008",
             "Invalid Interface Inherited",
-            "Runtime component class {0} cannot implement interface {1}, as the interface is not a valid Windows Runtime interface");
+            "Windows Runtime component class {0} cannot implement interface {1}, as the interface is not a valid Windows Runtime interface");
 
         public static DiagnosticDescriptor ClassConstructorRule = MakeRule(
-            "WME1099",
+            "CsWinRT1009",
             "Class Constructor Rule",
-            "Runtime component class {0} cannot have multiple constructors of the same arity {1}");
+            "Classes cannot have multiple constructors of the same arity in the Windows Runtime, class {0} has multiple {1}-arity constructors");
 
         public static DiagnosticDescriptor ParameterNamedValueRule = MakeRule(
-            "WME1092",
+            "CsWinRT1010",
             "Parameter Named Value Rule",
-            "The method {0} has a parameter named {1} which is the same as the default return value name. "
-            + "Consider using another name for the parameter or use the System.Runtime.InteropServices.WindowsRuntime.ReturnValueNameAttribute "
-            + "to explicitly specify the name of the return value.");
+            "The method {0} is used in the Windows Runtime and has a parameter named {1}." +
```

Comment:
Can we just merge both like The parameter name {1} in method {0} is the same as the return value used ...